k8s/prod/common/bar/tests-values.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: bitnami.com/v1alpha1
```

Comment:
prob want to update codeowners for this file pattern

Reply:
Yeah, even if I suspect it will have to be us anyway :-)

Reply:
i mean add us